src/Parse/Semvar.php

```diff
@@ -0,0 +1,229 @@
+<?php namespace Winter\Storm\Parse;
+
+/**
+ * Semvar parser to allow for version matching against Composer
+ * style rules
+ *
+ * @author Jack Wilkinson
+ */
+
+class Semvar
+{
+    const LOGICAL_OR = '||';
+
+    const OPERATOR_GTE   = 'greaterThanEqual';
+    const OPERATOR_GT    = 'greaterThan';
+    const OPERATOR_LTE   = 'lessThanEqual';
+    const OPERATOR_LT    = 'lessThan';
+    const OPERATOR_TILDY = 'tildy';
```

Comment:
Should be `OPERATOR_TILDE = 'tilde';`

Reply:
That's my dyslexia showing 😂. I'll change it in the next push :)